### DISCUSSION TITLE
Error (6275): Source "forge-std/Script.sol" not found: File not found

### DISCUSSION THREAD

**Discussion by gopalchandak-dev** [Cyfrin/security-and-auditing-full-course-s23#44] [Q&A] [answered]
I'm having trouble importing "forge-std" into my script and test, and despite doing a number of checks, I still can't figure out the problem. Despite watching numerous videos, I'm still unable to solve.

* lib folder is empty.
* I have rechecked using "foundryup".
* I have made another project, but still getting the same problem.

![Screenshot 2023-12-23 234859](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/141278896/e600f8a7-854a-4d78-b323-93fd9f8b0d6e)

**Comment by TilakMaddy** [comment]
tried `forge compile` ?

**Reply by gopalchandak-dev** [reply]
yes tried.

**Comment by cromewar** [comment]
Most of times a simple terminal re-start would help. Don't forget to compiler the code as @TilakMaddy suggested.

Anyways this question was asked a little time ago, @gopalchandak-dev were you able to solve this? If not, please let us know and we will provide support.

**Reply by gopalchandak-dev** [reply, marked as answer]
It's not working even after re-starting terminal. 

' ' '
### forge ERROR ---> forge install foundry-rs/forge-std@v1.5.3 --no-commit
' ' '
But, after running this in the terminal, now it's properly working.

**Reply by cromewar** [reply]
thanks for sharing @gopalchandak-dev
